src-tauri/Cargo.toml

```diff
@@ -66,19 +66,22 @@ features = [
 winreg = "0.10.1"
 
 [dependencies.tauri]
-features = ["clipboard-all", "dialog-all", "fs-all", "global-shortcut-all", "gtk-tray", "notification-all", "os-all", "path-all", "process-exit", "process-relaunch", "shell-all", "system-tray", "window-all", "updater"]
+features = ["clipboard-all", "dialog-all", "fs-all", "global-shortcut-all", "gtk-tray", "notification-all", "os-all", "path-all", "process-exit", "process-relaunch", "shell-all", "system-tray", "updater", "window-all"]
 version = "1.0.0-rc.11"
 
 [features]
 default = [ "custom-protocol" ]
 custom-protocol = [ "tauri/custom-protocol" ]
+opencl = ["subspace-farmer/opencl"]
```

Comment:
Did you confirm necessary OpenCL dependencies are actually in the `*.deb` package dependencies if you enable this feature though?

I don't see any related changes, but maybe Tauri is smart enough to do that automatically. Also I don't see CI pipeline changes, so this feature doesn't have any effect yet anyway.

Reply:
I did not confirm they are in the deb package :/ will try to check 
Updated CI pipeline 👍

Reply:
opencl library is not in the deb package as you guessed... ill look for a way to include them in the deb

Reply:
found the way to specify dependencies in tauri conf json, trying to figure out doing that conditionally for our case `#[cfg(feature = "opencl")]`